createWebNavigator.js

```diff
@@ -1,39 +1,38 @@
 import _ from 'underscore';
 import * as React from 'react';
-import {View, Pressable} from 'react-native';
+import {View, Dimensions} from 'react-native';
 import {useNavigationBuilder, createNavigatorFactory, StackRouter} from '@react-navigation/native';
+import RHPContainer from './components/RHPContainer';
+
+const isSmallScreenWidth = Dimensions.get('window').width <= 800;
+
 
 function CustomStackNavigator(props) {
     const {navigation, state, descriptors, NavigationContent} = useNavigationBuilder(StackRouter, {
       children: props.children,
       screenOptions: props.screenOptions,
       initialRouteName: props.initialRouteName,
     });
-
+    const isLHNVisible = (state.index === 0 && state.routes.length === 1) || !isSmallScreenWidth;
```

Comment:
**noting:** no need for this check here or `isLHNVisible` if we use the native stack nav for small screens. the LHN will always be visible in the custom web layout.